src/replace.js

```diff
@@ -0,0 +1,29 @@
+const { matchExpr, isStart, removeSlashWrapper } = require('./utils');
+
+function replaceByRegExp(inputExpr, word, str) {
+  if (inputExpr.length === 0 || inputExpr === '//') throw new Error('There is no RegExp. To solve it enter the RegExp');
+  if (str.length === 0) throw new Error('Text to search is empty. To solve this problem enter the text');
+  
+  let expr = removeSlashWrapper(inputExpr);
+  let resStr = str;
+  
+  if (isStart(expr[0])) {
```

Comment:
Do not call a function inside `if` statement. Move it out to another constant identifier